src/main/java/org/graylog/aws/inputs/flowlogs/FlowLogsInput.java

```diff
@@ -12,14 +14,14 @@
 import javax.inject.Inject;
 
 public class FlowLogsInput extends MessageInput {
-    private static final String NAME = "AWS FlowLogs Input";
+    private static final String NAME = "AWS CloudWatch Flow Logs";
```

Comment:
Tiny minor thing: I'd rename this to just "AWS Flow Logs", because CloudWatch is really just a transport and it could be a little confusing for users who think that this is Flow Logs *for* CloudWatch.